test/samples/dump.txt

```diff
@@ -0,0 +1,63 @@
+{HEADME.md
```

Comment:
Let'suse for isntance https://github.com/AnyFetch/ocr-hydrater.anyfetch.com/blob/master/test/samples/sample.png